mm2src/common/mm_error/mm_error.rs

```diff
@@ -0,0 +1,386 @@
+//! The MarketMaker2 unified error representation tracing an error path.
+//!
+//! # Tracing while chaining
+//!
+//! `MmError` supports several ways to trace the error path while different error types replace each other along the path.
+//!
+//! ## Operator `?`
+//!
+//! `MmError` main goal is to use the operator `?` to convert one `E1` type into another `E2` type
+//! and track the place where this conversation took place.
```

Comment:
I guess you meant `conversion` instead of `conversation` :slightly_smiling_face:

Reply:
Sure :D

Reply:
Done